src/database.ts

```diff
@@ -114,25 +114,24 @@ export const redshiftClusterMaintenanceSettings: ResourceValidationPolicy = {
             },
             preferredMaintenanceWindow: {
                 type: "string",
-                default: "Mon:09:30-Mon:10:00",
```

Comment:
I don't think we want to set a default value for this config property. It's optional -- if someone specifies it, then we will do additional checks.